code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -103,6 +103,53 @@
 	poison_per_bite = 5
 	move_to_delay = 5
 
+//vipers are the rare variant of the hunter, no damage but so much poison medical care will be needed fast. Low hp, high speed and no damage so hit and run tactics.
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper
+	desc = "Furry and black, it makes you shudder to look at it. This one has effervescent purple eyes."
+	icon_state = "hunter"
+	icon_living = "hunter"
+	icon_dead = "hunter_dead"
+	maxHealth = 40
+	health = 40
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	poison_per_bite = 30
+	move_to_delay = 4
+	poison_type = "polonium" //slow acting but this at 30u per bite will seriously mess you up, perfect viper poison
+	speed = 4
+
+//tarantulas are really tanky, regenerating (maybe), hulky monster but are also extremely slow, so.
+/mob/living/simple_animal/hostile/poison/giant_spider/tarantula
+	desc = "Furry and black, it makes you shudder to look at it. This one has abyssal red eyes."
+	icon_state = "guard"
+	icon_living = "guard"
+	icon_dead = "guard_dead"
+	maxHealth = 400 // woah nelly
+	health = 400
+	melee_damage_lower = 35
+	melee_damage_upper = 40
+	poison_per_bite = 0
+	move_to_delay = 8
+	speed = 3
+	status_flags = NONE
+	mob_size = MOB_SIZE_LARGE
+
+//midwives are the queen of the spiders, can send messages to all them and web faster. That rare round where you get a queen spider and turn your 'for honor' players into 'r6siege' players will be a fun one.
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife

```

Comment:
no name difference?

Reply:
no? midwife?

Reply:
i don't know what you mean here

Reply:
you didn't set a name var.

Reply:
why are we giving them special names? Oh whatever, it'll be kinda nice when it happens.

Reply:
You're the one giving them hierarchical structure, would make sense for the top dog to be a different name than the common peon.

Reply:
I wish I knew how to remove this entire section since it was changed :(

Reply:
i could change the pr then change it back that would close this right